src/leiningen/new/db.clj

```diff
@@ -8,11 +8,18 @@
     (some #{"+mongodb"} features) :mongo
     (some #{"+h2"} features) :h2))
 
+(defn select-sql-dsl [{:keys [features]}]
+  (cond
+    (some #{"+yesql"} features) :yesql
+    (some #{"+korma"} features) :korma))
```

Comment:
yeah that looks good, are you planning on changing anything else or should we merge it in?